jsonnet/kube-thanos/kube-thanos-store.libsonnet

```diff
@@ -187,6 +187,11 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
         maxChunksGetRangeRequests: 3,
         chunkObjectSizeTTL: '24h',
         chunkSubrangeTTL: '24h',
+        blocksIterTTL: '5m',
+        metafileExistsTTL: '2h',
+        metafileDoesntExistTTL: '15m',
+        metafileContentTTL: '24h',
+        metafileMaxSize: '1MiB',  // Equal to default MaxItemSize in memcached client.
```

Comment:
I think this isn't actually used (`metafile_max_size:` uses `m.maxItemSize` instead of `c.metafileMaxSize`).

Reply:
Yes, it's not used. I wanted to keep it to hint the user, but it was a bad idea. Removing it.